src/ontology/mondo.Makefile

```diff
@@ -681,6 +681,15 @@ test_owlaxioms:
 
 test: test_owlaxioms 
 
+.PHONY: test_obs_reason
+test_obs_reason:
+	echo "all obsolesence reasons should be typed as xsd:string"
+	! grep -E "property_value: IAO:0000231.*xsd:[^s]" mondo-edit.obo
+	echo "obsolesence reason does not seem to use the correct property"
+	! grep -E "\"terms merged\" xsd:" mondo-edit.obo | grep -v IAO:0000231
```

Comment:
@nicolevasilevsky note this check only covers the "terms merged" case, and does not check other obsolescence reasons. You can copy paste this line to capture other obsolescence reasons.

Reply:
updated